app/services/api/v3/actors/basic_attributes.rb

```diff
@@ -57,19 +62,34 @@ def summary
         NAMED_ATTRIBUTES = %w(forest_500 zero_deforestation).freeze
 
         def initialize_named_attributes
-          values =
-            NAMED_ATTRIBUTES.map do |name|
-              original_attribute = @chart_config.named_attribute(name)
-              next nil unless original_attribute
-
-              value = @values.get(
-                original_attribute.simple_type, original_attribute.id
-              )
-              next nil unless value
-
-              [name, value]
-            end
-          Hash[values.compact]
+          values = {header_attributes: {}}
+          NAMED_ATTRIBUTES.map do |name|
+            original_attribute = @chart_config.named_attribute(name)
+            next nil unless original_attribute
+
+            value = @values.get(
+              original_attribute.simple_type, original_attribute.id
+            )
+            next nil unless value
+
+            values[name.to_sym] = value
+
+            chart_attribute = @chart_config.named_chart_attribute(name)
+            values[:header_attributes][chart_attribute.identifier.to_sym] =
+              header_attributes(chart_attribute)
+          end
+          values
+        end
+
+        def header_attributes(attribute)
+          {
+            name: attribute.display_name,
```

Comment:
the name comes up empty for me for some reason

<img width="996" alt="Screenshot 2019-07-19 at 22 40 19" src="https://user-images.githubusercontent.com/134055/61567094-3d21d100-aa76-11e9-8043-c9d2d934cef4.png">

should be "Forest 500 Score" and "Zero Deforestation Commitment" respectively I think